docs/api-reference/tools.md

```diff
@@ -20,6 +20,16 @@
 ::: strands.tools.watcher
     options:
       heading_level: 2
+::: strands.tools.executors
```

Comment:
Do we have an issue tracking auto-generating these?  If not could you create one?

Also, add the issue to the docs paper-cuts